src/FulfillerMultiSelect.jsx

```diff
@@ -276,17 +340,19 @@ class FulfillerMultiSelect extends React.Component {
         )
     }
 
-    fulfillerValueRenderer(fulfiller) {
-        return this.formatTitle(fulfiller) || this.tt('misconfigured');
+    fulfillerValueRenderer (option) {
+        return this.formatTitle(option.fulfiller) || this.tt('misconfigured');
```

Comment:
We brought this up and supposedly you know if there is anything to be done about this. (in handleChange, too)

Reply:
Yes, so previously there were two types of the `options` the select component would work with. I had to unify these so that the select component can match the selected values to options. And hence some logic supporting the two types could have been retired here